spec/api/xml_spec.rb

```diff
@@ -79,6 +79,17 @@
           expect(result).to match(/<field name="id">1<\/field>/)
         end
 
+        # add a single hash ("doc")
+        it 'should create an add from a hash formatted for atomic updates' do
+          data = {
+            :id=>1,
```

Comment:
👌 ; I've changed the new spec to use ruby 1.8-style hashes (as much as it pained me to do...)